examples/docker/README.rst

```diff
@@ -50,6 +50,13 @@ matches the name of the image referenced in the ``MLproject`` file.
 
 Finally, run the example project using ``mlflow run examples/docker -P alpha=0.5``.
 
+.. note::
+    If running this example on M1 / M2 Mac, ensure that Docker Desktop is running and that you are
```

Comment:
```suggestion
    If running this example on Mac with Apple silicon, ensure that Docker Desktop is running and that you are
```

Can we use `Apple silicon` instead of M1/M2?